api/.docker/modify_path_then

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+export PATH=$PATH:$PWD/node_modules/.bin
```

Comment:
Not for now, but this is probably an indication that we should push these into npm scripts, hey?

Reply:
Hmm, I'd argue this is a "both/and" situation rather than an "either/or" one.  For example, a lot of node developers (myself included) put `./node_modules/.bin` on their `PATH` just to make it super easy to access CLI tools that are part of a particular project--even if they are exposed via a script entry in `package.json`, there are advantages/disadvantages to using each.

IMO we should give devs the ability to use whichever approach is most comfortable or advantageous for their particular use case, and in this case I think that means we should _both_ modify the `PATH` _and_ expose more tools via `script` entries.